ai_presenter/reader.py

```diff
@@ -18,7 +18,7 @@ def __init__(self, file_path):
             self.actors[a.name] = a
         for scene in self.data['scenes']:
             s = Scene(scene)
-            self.scenes[s.name] = s
+            self.scenes.append([s.name, s])
```

Comment:
You can use a Tuple ( https://docs.python.org/3/tutorial/datastructures.html#tuples-and-sequences ) or you can use a Class